README.md

```diff
@@ -135,10 +112,7 @@ const renderOpts = {
     }
 };
 
-server.render('template', context, renderOpts, (err, output) => {
-
-    // ...
-});
+const output = await server.render('template', context, renderOpts);
```

Comment:
@wswoodruff should/could we assume `server.render` will become `async` also?

Reply:
Yup! 👍